examples/http3-server.rs

```diff
@@ -549,6 +580,39 @@ fn build_response(
     (headers, body)
 }
 
+/// Handles newly writable streams.
+fn handle_writable(client: &mut Client, stream_id: u64) {
+    let conn = &mut client.conn;
+    let http3_conn = &mut client.http3_conn.as_mut().unwrap();
+
+    debug!("{} stream {} is writable", conn.trace_id(), stream_id);
+
+    if !client.partial_responses.contains_key(&stream_id) {
```

Comment:
should this check be moved to the top of the function? That would prevent some unneeded prints, unless you specifically want to see when the transport can write to a stream but the app has nothing to write...

Reply:
I left the log specifically outside the check as I thought it would be useful for debugging (e.g. stream is writable but we don't have any data to send).

In practice I never used it though, so I don't really care that much.